store/tikv/lock_resolver.go

```diff
@@ -566,7 +583,13 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 			return status, err
 		}
 		status.action = cmdResp.Action
-		if cmdResp.LockTtl != 0 {
+		if cmdResp.LockInfo != nil {
+			status.secondaries = cmdResp.LockInfo.Secondaries
+			status.useAsyncCommit = cmdResp.LockInfo.UseAsyncCommit
+		}
+		if status.useAsyncCommit {
+			status.commitTS = cmdResp.LockInfo.MinCommitTs
```

Comment:
Should set `status.ttl = 0` when `lr.store.GetOracle().IsExpired(txnID, cmdResp.LockTtl)` is true.

Reply:
aiui, status.ttl is 0 by default, so we should set it to cmdResp.LockTtl when that condition is *not* true